src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue

```diff
@@ -188,16 +189,26 @@ const errorClass = computed(() => (props.invalid
   : []));
 const content = ref<InstanceType<typeof component.value> | null>(null);
 defineExpose({ content });
-const suffix = ref<ComponentPublicInstance | null>(null);
-const suffixSize = ref({
-  '--_label-suffix-width': '0',
-  '--_label-suffix-height': '0',
+const suffixTemplateRefs = ref<ComponentInstance<typeof UiListItemSuffix> | null>(null);
+const suffixEl = computed(() => (suffixTemplateRefs.value?.listItemSuffixTemplateRefs?.$el));
+const suffixSize = computed(() => {
+  let width = 0;
+  let height = 0;
+  if (suffixEl.value) {
+    const size = suffixEl.value.getBoundingClientRect();
+    width = size.width;
+    height = size.height;
+  }
+  return {
+    '--_label-suffix-width': `${width}px`,
+    '--_label-suffix-height': `${height}px`,
+  };
 });
 const handleInfoFocus = (event: KeyboardEvent) => {
   if (event.key !== 'ArrowRight') return;
-  if (suffix.value?.$el) {
+  if (suffixEl.value) {
```

Comment:
```suggestion
  if (!suffixEl.value) return;
```
and then without indentation, wdyt? :)